packages/nitro-protocol/readme.md

```diff
@@ -101,3 +97,11 @@ See https://docusaurus.io/docs/en/installation for more information.
 NB: you may run into difficulty running `docgen / solidoc` if you have the native solc compiler installed at the incorrect version number. You may refer to the circle `config.yml` at the monorepo root to check which version is being used as a part of our continuous integration.
 
 To add a new version of the docs, follow the instructions at https://docusaurus.io/docs/en/tutorial-version. We try to keep the documentation version in sync with the @statechannels/nitro-protocol npm package.
+
+## Deploying contracts to a testnet
+
+The current [hardhat](https://www.npmjs.com/package/hardhat-deploy) config works with Rinkeby testnet. It can be added on to for any other network.
+
+```
+INFURA_TOKEN=[your token here] RINKEBY_DEPLOYER_PK=[private key used for rinkeby deploy] npx hardhat deploy --network rinkeby
```

Comment:
Could we use `yarn` instead of `npx`?

Reply:
Good call! Do you mind making the change in your PR https://github.com/statechannels/statechannels/pull/2893?